Server/src/main/java/Database/TheAuPair/Models/Au_Pair.java

```diff
@@ -0,0 +1,120 @@
+package Database.TheAuPair.Models;
+
+import org.springframework.data.annotation.Id;
+import org.springframework.data.mongodb.core.mapping.Document;
+import org.springframework.data.mongodb.core.mapping.Field;
+
+@Document("Au-Pair")
+public class Au_Pair
+{
+    @Id
+    private String id;
+
+    @Field("rating")
+    private double rating;
+    @Field("pay_rate")
```

Comment:
Same here,  please confirm

Reply:
This is the pay rate (hourly rate) that an au pair will set up for their account which will be used to determine the au pairs salary.  I think this is fine.